x-pack/plugins/aiops/public/components/log_categorization/category_table/category_table.tsx

```diff
@@ -0,0 +1,270 @@
+      style: { display: 'block' },
+      render: (examples: string[]) => (
+        <div style={{ display: 'block' }}>
+          {examples.map((e) => (
+            <>
+              <EuiText size="s">
+                <EuiCode language="log" transparentBackground>
+                  {e}
+                </EuiCode>
+              </EuiText>
+              <EuiSpacer size="s" />
+            </>
+          ))}
+        </div>
+      ),
+    },
+    {
+      name: 'Actions',
+      width: 60,
+      actions: [
+        {
+          name: i18n.translate('xpack.aiops.logCategorization.showInDiscover', {
+            defaultMessage: 'Show these in discover',
+          }),
+          description: i18n.translate('xpack.aiops.logCategorization.showInDiscover', {
+            defaultMessage: 'Show these in discover',
+          }),
+          icon: 'discoverApp',
+          type: 'icon',
+          onClick: (category) => openInDiscover(QUERY_MODE.INCLUDE, category),
+        },
+        {
+          name: i18n.translate('xpack.aiops.logCategorization.filterOutInDiscover', {
+            defaultMessage: 'Filter out in discover',
+          }),
+          description: i18n.translate('xpack.aiops.logCategorization.filterOutInDiscover', {
+            defaultMessage: 'Filter out in discover',
+          }),
+          icon: 'filter',
+          type: 'icon',
+          onClick: (category) => openInDiscover(QUERY_MODE.EXCLUDE, category),
+        },
+        // Disabled for now
+        // {
+        //   name: i18n.translate('xpack.aiops.logCategorization.openInDataViz', {
+        //     defaultMessage: 'Open in data visualizer',
+        //   }),
+        //   icon: 'stats',
+        //   type: 'icon',
+        //   onClick: () => {},
+        // },
+      ],
+    },
+  ] as Array<EuiBasicTableColumn<Category>>;
+
+  const selectionValue: EuiTableSelectionType<Category> | undefined = {
+    selectable: () => true,
+    onSelectionChange: (selectedItems) => setSelectedCategories(selectedItems),
+  };
+
+  const getRowStyle = (category: Category) => {
+    if (
+      pinnedCategory &&
+      pinnedCategory.key === category.key &&
+      pinnedCategory.key === category.key
+    ) {
+      return {
+        backgroundColor: 'rgb(227,240,249,0.37)',
+      };
+    }
+
+    if (
+      selectedCategory &&
+      selectedCategory.key === category.key &&
+      selectedCategory.key === category.key
+    ) {
+      return {
+        backgroundColor: euiTheme.euiColorLightestShade,
+      };
+    }
+
+    return {
+      backgroundColor: 'white',
+    };
+  };
+
+  return (
+    <>
+      {selectedCategories.length > 0 ? (
+        <>
+          <EuiFlexGroup>
+            <EuiFlexItem grow={false}>
+              <EuiButton size="s" onClick={() => openInDiscover(QUERY_MODE.INCLUDE)}>
+                <FormattedMessage
+                  id="xpack.aiops.logCategorization.showInDiscover"
+                  defaultMessage="Show these in discover"
```

Comment:
As above, should be capitalized Discover for these labels.

Reply:
Updated in https://github.com/elastic/kibana/pull/139005/commits/90b39a5ac04264c97ad812521e40c56d23477963